FactoryMethod/Projects/Logistics/src/SeaLogistics.h

```diff
@@ -0,0 +1,25 @@
+//
+// Created by @PolinaShlepakova on 17.02.2019.
+//
+
+#ifndef LOGISTICS_SEALOGISTICS_H
+#define LOGISTICS_SEALOGISTICS_H
+
+#include "Ship.h"
+#include "Cargo.h"
+#include "Destination.h"
+#include "Logistics.h"
+
+class SeaLogistics : public Logistics {
+public:
+	SeaLogistics();
+	~SeaLogistics() override;
+	SeaLogistics(const SeaLogistics&) = delete;
+	SeaLogistics& operator=(const SeaLogistics&) = delete;
+
+	Transport* createTransport() const;
```

Comment:
@PolinaShlepakova,
Why don't you use `override` modifier in `createTransport()` method?

Reply:
It's a typo, c++ doesn't require it. Thanks for mentioning!